packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -778,6 +778,30 @@ let getSingleUseLegacyLogCallback = (logger: ITelemetryLoggerExt, type: string)
 	};
 };
 
+/**
+ * Load the stores from a snapshot and returns the runtime.
+ * @param params - An object housing the runtime properties:
```

Comment:
Nit: I would recommend adding an actual type for the params here. Makes the documentation look significantly easier (you can more each property doc to the actual property on the type, etc.). Having types like `LoadContainerRuntimeProps` is a pretty common pattern, and I personally think we should err on the side of using it over inline types.

Reply:
i just copied this from loadRuntime. i can add a new type, and expand the docs.

Reply:
@Josmithr added a new interface, and added some comments